src/server_manager/web_app/ui_components/app-root.js

```diff
@@ -720,9 +734,7 @@ export class AppRoot extends mixinBehaviors
   }
 
   handleRegionSelected(/** @type {Event} */ e) {
-    this.fire('SetUpServerRequested', {
-      regionId: e.detail.selectedRegionId,
-    });
+    this.fire('SetUpServerRequested', e.detail);
```

Comment:
This is harder to read. I no longer know what this method fires. Could you build the details object here?

Reply:
Done.  I also changed the event name to emphasize that it is only for DO.